package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "nativescript-drop-down",
-  "version": "4.0.1",
+  "version": "5.0.0",
```

I think no need to up it with a major versiona and we can release this under 4.1.0
```suggestion
  "version": "4.1.0",
```

---

As mentioned in the main post,

>I'd suggest doing so as a breaking change release as anyone using this is probably expecting matching styles but now a <DropDownList> child of <DropDown> is required to style the actual list items.

If I was an existing user of this expecting the functionality of styles on the `DropDown` to apply to the list I'd probably want to know that that wasn't the case after updating.

---

Well not a problem to release a breaking change in a minor version bump :)

Also I think we can avoid the breaking change so that if there is no drop down style defined to use the parent style.